app.config.js

```diff
@@ -169,7 +149,7 @@ module.exports = function () {
               organization: 'blueskyweb',
               project: 'react-native',
               release: VERSION,
-              dist: `${PLATFORM}.${VERSION}.${DIST_BUILD_NUMBER}`,
+              dist: `${PLATFORM}.${VERSION}.${new Date().toISOString()}`,
```

Comment:
Sorry none of this has comments on it, but we need to keep these values in sync with our Sentry config in order for source maps to work. Let's chat in Slack 👍

Reply:
I think this will cover it. Going to download the IPA and install to make sure it sets the version correctly.

Reply:
Bundle looks good, just check Sentry and we should be good to go.